Interop/StubGenerator/src/main/kotlin/org/jetbrains/kotlin/native/interop/gen/MappingBridgeGeneratorImpl.kt

```diff
@@ -57,8 +57,10 @@ class MappingBridgeGeneratorImpl(
             is RecordType -> {
                 val mirror = mirror(declarationMapper, returnType)
                 val tmpVarName = kniRetVal
-                builder.out("val $tmpVarName = nativeHeap.alloc<${mirror.pointedType.render(builder.scope)}>()")
-                builder.pushBlock("try {", free = "finally { nativeHeap.free($tmpVarName) }")
+                // We clear in the finally block.
+                builder.pushMemScoped(explicitClear = false)
```

Comment:
Using `nativeHeap` as before eliminates the necessity to have `explicitClear = false` mode.